source/standards/reliability-engineering.html.md.erb

```diff
@@ -0,0 +1,23 @@
+---
+title: Reliability Engineering
+expires: 2018-12-01
+---
+
+# <%= current_page.data.title %>
+
+Reliability Engineering provide a shared platform to GDS teams made up of tools and services to set up and maintain a service, for example by:
+
+* supporting the Amazon Web Services (AWS) and the GOV.UK PaaS infrastructure
+*  using [Logit][] to provision, manage and ensure availability of team's logging infrastructure
+*  running an operational metrics service using [Prometheus][]
+*  managing a shared base AWS account for GDS teams
+
+[Reliability Engineering documentation][] explains the team's full range of services and how you can access and use them.
```

Comment:
This may be a little bit easier to parse:

> You can read more about Reliability Engineering in [our documentation](https://reliability-engineering.cloudapps.digital/).